crates/holochain_state/src/query.rs

```diff
@@ -704,6 +747,11 @@ impl<'borrow, 'txn> Store for DbScratch<'borrow, 'txn> {
         }
     }
 
+    fn get_warrants_for_basis(&self, hash: &AnyLinkableHash) -> StateQueryResult<Vec<Warrant>> {
+        // The scratch will never
+        self.txns.get_warrants_for_basis(hash)
+    }
```

Comment:
```suggestion
    fn get_warrants_for_basis(&self, hash: &AnyLinkableHash) -> StateQueryResult<Vec<Warrant>> {
        // The scratch will never contain warrants, since they are not committed to chain
        self.txns.get_warrants_for_basis(hash)
    }
```